html/form/the-disabled-attribute.html

```diff
@@ -0,0 +1,43 @@
+<!doctype html>
+<title>form disabled attribute</title>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/form-elements.html#attr-form-disabled">
+<link rel="help" href="https://github.com/whatwg/html/issues/3682">
+<link rel="author" href="mailto:cyril.auburtin@gmail.com" title="Cyril Auburtin">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<form>
+  <input name="foo">
+  <input name="sb" type="submit">
+</form>
+<form disabled>
+  <input name="foo">
+  <input name="sb" type="submit">
+</form>
+<script>
+  test(() => {
+    const [form1, form2] = document.forms;
+
+    form1.disabled = true;
+    assert_true(form1.disabled);
+
+    form2.focus();
+    assert_true(form2.disabled);
+    assert_equals(document.activeElement, document.body);
+
+    const submitHandler = e => {
```

Comment:
Please use `t.unreached_func("...")` (where `t` is the first argument to the function you pass to `test()`).